migrations/versions/d1b71925b9e3_add_codename.py

```diff
@@ -0,0 +1,28 @@
+"""empty message
+
+Revision ID: d1b71925b9e3
+Revises: 0320de7481ac
+Create Date: 2023-04-12 18:12:13.413661
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'd1b71925b9e3'
+down_revision = '0320de7481ac'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
```

Comment:
Could these comments and the ones below be removed as they don't add any useful information?